mappings/net/minecraft/client/render/model/BakedModel.mapping

```diff
@@ -12,6 +12,9 @@ CLASS net/minecraft/class_1087 net/minecraft/client/render/model/BakedModel
 	METHOD method_4708 useAmbientOcclusion ()Z
 	METHOD method_4709 getTransformation ()Lnet/minecraft/class_809;
 	METHOD method_4710 getOverrides ()Lnet/minecraft/class_806;
-	METHOD method_4711 getSprite ()Lnet/minecraft/class_1058;
+	METHOD method_4711 getModelIcon ()Lnet/minecraft/class_1058;
```

Comment:
> the other methods in the class have `Model` in the name, like `getModel`

but not for this one

```suggestion
	METHOD method_4711 getIcon ()Lnet/minecraft/class_1058;
```

Reply:
the `Model` here is here for a different reason, it's there to reduce ambiguity. Just getIcon by itself is too terse imo

Reply:
Sure, we can let people comment on this suggestion later